### PR TITLE
Gfm 0.29

### PR DESCRIPTION
**Pull request by UziTech**
**Marked version:** master

**Markdown flavor:** GitHub Flavored Markdown

## Description

Update gfm specs to 0.29

## Contributor

- [x] Test(s) exist to ensure functionality and minimize regression

## Committer

In most cases, this should be a different person than the contributor.

- [x] Draft GitHub release notes have been updated.
- [x] CI is green (no forced merge required).
- [x] Merge PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
   ],
   "devDependencies": {
     "@markedjs/html-differ": "^2.0.1",
+    "cheerio": "^1.0.0-rc.3",
```
Comment (styfle) [inline package.json]: rc3?

File: test/specs/gfm/gfm.0.29.json [hunk 2]
```diff
@@ -1,153 +1,147 @@
 [
   {
     "section": "Tables",
-    "html": "<table>\n<thead>\n<tr>\n<th>foo</th>\n<th>bar</th>\n</tr>\n</thead>\n<tbody>\n<tr>\n<td>baz</td>\n<td>bim</td>\n</tr></tbody></table>",
+    "html": "<table>\n<thead>\n<tr>\n<th>foo</th>\n<th>bar</th>\n</tr>\n</thead>\n<tbody>\n<tr>\n<td>baz</td>\n<td>bim</td>\n</tr>\n</tbody>\n</table>",
     "markdown": "| foo | bar |\n| --- | --- |\n| baz | bim |",
-    "example": 191
+    "example": "198"
```
Comment (styfle) [inline test/specs/gfm/gfm.0.29.json]: Does it matter if `example` is a string or a number?

Comment (UziTech) [reply test/specs/gfm/gfm.0.29.json]: probably not but I fixed it so they are ints

File: package.json (hunk 1 above)
Comment (UziTech) [reply package.json]: Should I change it to v0.22.0? either one works for what we need from it.

Comment (styfle) [reply package.json]: It looks like v0.22.0 is 3 years old so lets keep it rc3

**Pull request merged**